### DISCUSSION TITLE
Good First Issue

### DISCUSSION THREAD

**Discussion by Anthony-Eid** [Whisky-App/Whisky#108] [General]
Hey, I'm a young development (just finished my last computer science classes in undergrad) and would love to contribute to this project. I don't have much experience in swift yet so I was hoping someone could point out some good first issues for me to take care of. 

I just wanna learn and get more involved within the open source community. I could start some documentation, fix compiler warnings if there are any, as well as anything that you would recommend. Is there anything in particular that I should start doing?

**Comment by dristanta-silwal** [comment]
Hey there! That's awesome to hear that you're eager to contribute to this project and get more involved in the open source community. As a newcomer to Swift, there are several areas where you can make a meaningful contribution. Starting with documentation is a great idea—it helps improve the project's usability and accessibility. You could review existing documentation, suggest improvements, or even create new guides or tutorials for specific features or use cases. Additionally, fixing compiler warnings is a valuable task that can enhance the codebase's cleanliness and maintainability. Another suggestion would be to look for "good first issues" labeled specifically for beginners. These are typically small and approachable tasks that can help you get familiar with the codebase and the development process. Remember to reach out to the project maintainers and community if you have any questions or need guidance. Happy contributing and learning!
